dd-java-agent/instrumentation/mongo-3.1/src/main/java/datadog/trace/instrumentation/mongo/MongoClientDecorator.java

```diff
@@ -70,6 +71,7 @@ public Span onStatement(final Span span, final BsonDocument statement) {
     final String mongoCmd = scrubbed.toString();
 
     span.setTag(DDTags.RESOURCE_NAME, mongoCmd);
+    Tags.DB_STATEMENT.set(span, mongoCmd);
```

Comment:
Is this not already done by the subsequent `DatabaseClientDecorator` `onStatement`?